test/sensitivities/linalg/factorization/svd.jl

```diff
@@ -43,4 +43,14 @@
         @test Nabla.eyesubx!(copy(X)) ≈ I - X
         @test Nabla.add!(copy(X), Y) ≈ X + Y
     end
+
+    @testset "Tape updating from multiple components" begin
+        ∇f = ∇() do X
+            U, S, V = svd(X)
+            Y = U * Diagonal(S) * V'
+            sum(Y)
+        end
+        X = [1.0 2.0; 3.0 4.0]
```

Comment:
Since this is being implemented to check the case in which the components of the SVD have different sizes, it would be nice to test with a non-square `X`, such that we trigger that condition.

Reply:
Good idea, done. :+1: